### ISSUE TITLE
DependencyNotFoundError

### ISSUE THREAD

**Issue by simonghales** [RenaudRohlinger/r3f-perf#1]
The issue:

When installing r3f-perf and importing the <Perf/> component, the following error occurs:

```
DependencyNotFoundError

Could not find dependency: '@react-icons/all-files' relative to '/node_modules/r3f-perf/dist/r3f-perf.cjs.development.js'
```

Example Codesandbox: https://codesandbox.io/s/upbeat-pasteur-mvx5d?file=/src/App.js

I suspect that the dev dependencies aren't being installed by default when the package is downloaded, but I could be wrong. Happy to have a closer look tomorrow (it's 2am so I need to get some sleep.)

Cheers

**Comment by RenaudRohlinger** [comment]
@simonghales Oops indeed I forgot. It's fixed with 0.3.2!

https://codesandbox.io/s/clever-butterfly-ksk62?file=/public/index.html

**Comment by simonghales** [comment]
Great, thanks for the speedy resolution :) looks good!
